### ISSUE TITLE
Blender 2.93.2 - Tissue addon not loading?

### ISSUE THREAD

**Issue by enzyme69** [alessandro-zomparelli/tissue#121]
Hi Alessandro, with 2.93.2, the addon does not seem to load and Numba cannot be installed from addon preferences.

**Comment by enzyme69** [comment]
Actually the one from "release" seems to work, but the "Numba" still does not install
https://github.com/alessandro-zomparelli/tissue/releases/tag/v0-3-50

**Comment by JoelGodin** [comment]
Yes, as in many previous issues it has been stated, the bundled Tissue is not the one to use.

As for Numba are you on a Mac?

**Comment by enzyme69** [comment]
I am using Numba on macOS Monterey

**Comment by JoelGodin** [comment]
For some reason it doesn't install on Mac.
I have an old High Sierra and a new 2021 M1 with Big Sur and neither one
installs.
